cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -739,6 +739,8 @@ spec:
             {{- if .Values.global.grafana }}
             - name: GRAFANA_ENABLED
               value: "{{ template "cost-analyzer.grafanaEnabled" . }}"
+            - name: LOG_LEVEL
+              value: {{ .Values.kubecostModel.logLevel | default "info" }}
```

Comment:
@mittal-ishaan I'd prefer we not set any default values in the template files. Instead I prefer all defaults be set via the `values.yaml` file (which you've already done).